### PR TITLE
Logistic Regression Support for DataGen

### PR DESCRIPTION
**Pull request by Er1cCheng**
Add in support for generate random logistic regression binary label used for benchmark. Minor bug fix for argument parsing.

### CODE REVIEW

**Comment by lijinf2** [comment]
build

File: python/benchmark/gen_data_distributed.py [hunk 1]
```diff
@@ -448,6 +451,15 @@ def make_regression_udf(iter: Iterable[pd.DataFrame]) -> Iterable[pd.DataFrame]:
                 if noise > 0.0:
                     y += generator_p.normal(scale=noise, size=y.shape)
 
+                # Logistric Regression sigmoid and sample
+                if logistic_regression:
+                    if use_cupy:
+                        prob = 1 - 1 / (1 + cp.exp(-y))
```
Comment (lijinf2) [inline python/benchmark/gen_data_distributed.py:457]: It seems y gets larger, prob will be smaller. Maybe not an issue. PR looks good.

Comment (eordentlich) [reply python/benchmark/gen_data_distributed.py:457]: Yes if this prob of selecting 1, it should be 1/(1 + exp(-y))

File: .github/workflows/blossom-ci.yml [hunk 2]
```diff
@@ -42,11 +42,10 @@ jobs:
       rongou,\
       wjxiz1992,\
       GaryShen2008,\
-      pxLi,\
       NvTimLiu,\
       YanxuanLiu,\
       zhanga5,\
-      rishic3,\
+      Er1cCheng,\
```
Comment (eordentlich) [inline .github/workflows/blossom-ci.yml:48]: Strange.   This change was already merged in.   Not clear why it is showing as diff.

**Comment by Er1cCheng** [comment]
build

**Comment by Er1cCheng** [comment]
build

File: .github/workflows/blossom-ci.yml (hunk 2 above)
Comment (Er1cCheng) [reply .github/workflows/blossom-ci.yml:48]: I think maybe I did the merge in a wrong way so it's pulled as a diff.

**Comment by Er1cCheng** [comment]
build

**Comment by lijinf2** [comment]
build

**Review by lijinf2 — APPROVED** [review]

**Pull request merged**